dspace/config/submission-forms.xml

```diff
@@ -764,12 +1289,469 @@
                 <displayed-value>Turkish</displayed-value>
                 <stored-value>tr</stored-value>
             </pair>
+            <pair>
+                <displayed-value>Portuguese</displayed-value>
+                <stored-value>pt</stored-value>
+            </pair>
+            <pair>
+                <displayed-value>Turkish</displayed-value>
+                <stored-value>tr</stored-value>
```

Comment:
Turkish seems to be listed twice (by accident).  I'm fine with it being listed last, but we should remove the Turkish entry just before Portuguese

Reply:
Thank you @tdonohue for spotting this! Fixed!